ui/package.json

```diff
@@ -104,12 +103,11 @@
     "ember-cli-sri": "meirish/ember-cli-sri#rooturl",
     "ember-cli-string-helpers": "6.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-composable-helpers": "^4.3.0",
-    "ember-concurrency": "^2.1.2",
+    "ember-composable-helpers": "5.0.0",
+    "ember-concurrency": "2.3.4",
     "ember-copy": "2.0.1",
     "ember-d3": "^0.5.1",
-    "ember-data": "~4.4.0",
-    "ember-data-model-fragments": "5.0.0-beta.5",
+    "ember-data": "~4.5.0",
```

Comment:
I ran into [an issue](https://github.com/emberjs/data/issues/7997) where an error was thrown when doing `yield model.save();` in a concurrency task which prompted the bump to `4.5`.